kahuna/public/stylesheets/main.css

```diff
@@ -2096,3 +2097,63 @@ FIXME: what to do with touch devices
 .radio-list__label-value {
     padding: 5px 10px;
 }
+
+
+
+@media print {
+
+    * {
```

Comment:
This is a very expensive selector (selects every single element on the page). I don't know enough of how `@media` works to know when it is applied, but is there another way?